### PR TITLE
Fix empty change utxo

### PR DESCRIPTION
**Pull request by gege251**
Solves https://github.com/mlabs-haskell/CardStarter-bulk-airdrop-script/issues/2

### CODE REVIEW

File: src/FakePAB/PreBalance.hs [hunk 1]
```diff
@@ -151,7 +151,9 @@ balanceNonAdaOuts changeAddr utxos tx =
             txOuts
           (txOut@TxOut {txOutValue = v} : txOuts, txOuts') ->
             txOut {txOutValue = v <> nonAdaChange} : (txOuts <> txOuts')
-   in Right $ tx {txOutputs = outputs}
+   in if not (Value.isZero nonAdaChange)
+        then Right $ tx {txOutputs = outputs}
+        else Right tx
```
Comment (samuelWilliams99) [inline src/FakePAB/PreBalance.hs:156]:
```suggestion
   in Right $ if null nonAdaChange
        then tx
        else tx {txOutputs = outputs}
```
Thoughts on this?

File: src/FakePAB/PreBalance.hs [hunk 2]
```diff
@@ -43,8 +44,7 @@ preBalanceTx ::
 preBalanceTx minLovelaces fees utxos changeAddr tx =
   addTxCollaterals utxos tx
     >>= balanceTxIns utxos minLovelaces fees
```
Comment (Benjmhart) [inline src/FakePAB/PreBalance.hs:46]: prefer `=<<` and `<$>`

Comment (Benjmhart) [reply src/FakePAB/PreBalance.hs:46]: prefer do notation over `=<<`

Comment (samuelWilliams99) [reply src/FakePAB/PreBalance.hs]: Have used do notication

**Pull request merged**